analyzer/evmtokens/evm_tokens.go

```diff
@@ -0,0 +1,219 @@
+package evmtokens
```

Comment:
nit: would be nice to have the file and the folder have the same name; I slightly prefer `evm_tokens` over `evmtokens`, but either is fine.

Reply:
our org hasn't been using underscores in package names \:

Reply:
hm, package names in oasis core:

- gen_vectors
- example_signer_plugin
- gen_account_vectors

- pluginsigner
- remotesigner
- fixgenesis
- consensusrpc
- configparser
- testdata
- policywatcher
- enclaverpc
- keymanager
- txsource
- supplementarysanity
- roothash
- testvectors
- prettyprint
- workerpool
- accessctl
- tuplehash
- mathrand
- keyformat

- extract-metrics

packages names in runtime sdk:

- consensusaccounts

Reply:
https://google.github.io/styleguide/go/decisions#package-names oh I found this, maybe we've been following some stuff from Google's style guide for things where we had been indifferent

Reply:
resolving for merge

Reply:
Thanks for the pointer. Interesting; the styleguide just asserts "no underscores", and links to a Go(ogle) blog post that also just asserts "no underscores" without further justification.